examples/code/integrations/salesforce/config_provider.engine.yaml

```diff
@@ -0,0 +1,30 @@
+auth:
+  providers:
+    - id: salesforce
+      description: "Custom Salesforce provider"
+      enabled: true
+      type: oauth2
+      client_id: ${env:SALESFORCE_CLIENT_ID}
+      client_secret: ${env:SALESFORCE_CLIENT_SECRET}
+      oauth2:
+        authorize_request:
+          endpoint: 'https://{salesforce-org-subdomain}.my.salesforce.com/services/oauth2/authorize'
```

Comment:
Feel like this is going to mess people up. Since we use templates in the rest of this like '{{client_id}}' i assumed {salesforce-org-subdomain} was a template and it gave me an error I had to think about for a sec. 

Feel like someone unfamiliar is going to overlook that

Reply:
that makes sense.. perhaps we can use brackets? or asterisks? do you have a good idea?

Reply:
changed to simply `https://salesforce-org-subdomain.my.salesforce.com/services/oauth2/authorize`

in the docs, every time the domain is mentioned, I remind the reader to replace `salesforce-org-subdomain` with the actual one